aspen/http/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+:mod:`http`
+===========
+
+Aspen doesn't implement all of HTTP, only things related to filesystem
+dispatch.
+
+.. automodule:: aspen.http.mapping
+.. automodule:: aspen.http.request
+.. automodule:: aspen.http.resource
+
+"""
```

Comment:
> it looks like we can kill `Querystring` entirely

Actually, that may not be true. Even with `Querystring, we could combine all three `http` modules into a single file, and it'd only be ~300 lines.